<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTreeWidget" name="treeWidget_file">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
        <widget class="QTabWidget" name="editorTabWidget">
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>GDB Output</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPlainTextEdit" name="logView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Program Console</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <widget class="ConsoleWidget" name="targetOutputView">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>297</width>
               <height>10</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="autoWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="varWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="baseSize">
         <size>
          <width>80</width>
          <height>1</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_threads">
         <attribute name="title">
          <string>Threads</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTreeWidget" name="treeWidget_threads">
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_breakpoints">
         <attribute name="title">
          <string>Breakpoints</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTreeWidget" name="treeWidget_breakpoints">
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="allColumnsShowFocus">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_stack">
         <attribute name="title">
          <string>Stack</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="treeWidget_stack">
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuExecution">
    <property name="title">
     <string>Execution</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionStep_In"/>
    <addaction name="actionStep_Out"/>
    <addaction name="actionContinue"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewStack"/>
    <addaction name="actionViewBreakpoints"/>
    <addaction name="actionViewThreads"/>
    <addaction name="separator"/>
    <addaction name="actionViewWatch"/>
    <addaction name="actionViewAutoVariables"/>
    <addaction name="separator"/>
    <addaction name="actionViewTargetOutput"/>
    <addaction name="actionViewGdbOutput"/>
    <addaction name="actionViewFileBrowser"/>
    <addaction name="separator"/>
    <addaction name="actionDefaultViewSetup"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExecution"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionNext"/>
   <addaction name="actionStep_In"/>
   <addaction name="actionStep_Out"/>
   <addaction name="actionContinue"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/res/quit.png</normaloff>:/images/res/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/res/step_over.png</normaloff>:/images/res/step_over.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionStep_In">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/res/step_into.png</normaloff>:/images/res/step_into.png</iconset>
   </property>
   <property name="text">
    <string>Step In</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/res/stop.png</normaloff>:/images/res/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/res/continue.png</normaloff>:/images/res/continue.png</iconset>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/res/restart.png</normaloff>:/images/res/restart.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStep_Out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/res/step_out.png</normaloff>:/images/res/step_out.png</iconset>
   </property>
   <property name="text">
    <string>Step Out</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionViewStack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stack</string>
   </property>
  </action>
  <action name="actionViewBreakpoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Breakpoints</string>
   </property>
  </action>
  <action name="actionViewThreads">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Threads</string>
   </property>
  </action>
  <action name="actionViewWatch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Watch Variables</string>
   </property>
  </action>
  <action name="actionViewAutoVariables">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Variables</string>
   </property>
  </action>
  <action name="actionViewTargetOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Program Console</string>
   </property>
  </action>
  <action name="actionViewGdbOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GDB Output</string>
   </property>
  </action>
  <action name="actionViewFileBrowser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File Browser</string>
   </property>
  </action>
  <action name="actionDefaultViewSetup">
   <property name="text">
    <string>Restore Default View</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>consolewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeWidget_file</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
